src/components/MessageCenter/NotificationDrawer.tsx

```diff
@@ -63,14 +63,41 @@ type NotificationWrapperPropsType = {
   onClick: (message: NotificationMessage) => void;
 };
 
-class NotificationWrapper extends React.PureComponent<NotificationWrapperPropsType, StatelessType> {
+type NotificationWrapperStateType = {
+  isExpanded: boolean;
+};
+
+class NotificationWrapper extends React.PureComponent<NotificationWrapperPropsType, NotificationWrapperStateType> {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isExpanded: false
+    };
+  }
+
+  onToggle = () => {
+    this.setState({
+      isExpanded: !this.state.isExpanded
+    });
```

Comment:
Documentation at https://reactjs.org/docs/state-and-lifecycle.html#state-updates-may-be-asynchronous suggests that this should be changed to:
```js
this.setState((state) => ({
  isExpanded: !state.isExpanded
}));
```

Reply:
@israel-hdez  thanks for the tip!  PR has been updated.